src/services/httpService.js

```diff
@@ -31,7 +31,13 @@ export class HttpService {
     }
   }
 
-  async post(uri, params) {
-    return this.instance.post(uri, params);
+  async post(uri, data, params) {
+    return this.instance.post(uri, data, params);
+  }
+
+  static async getImageStream(imageUrl) {
```

Comment:
async? but no await?

Reply:
there is no observable difference between `return` and `return await`. Both versions of delay1Second have the exact same observable behavior and both of them will return a `promise`, the difference will be observed when you have a `try catch` block , you need to await in order the allow `throw` error to catch it, but once you are returning a `promise` there's no difference, take a look to the conclusion of this article.
https://dmitripavlutin.com/return-await-promise-javascript/